### PR TITLE
Use native Portage tools by default

### PR DESCRIPTION
**Pull request by td5038**
This will allow people who have moved away from layman to still use sysget to sync

### CODE REVIEW

**Comment by cvengler** [comment]
@nekobbbbbbit could you take a look this? Maybe I can do a new release after a **four year** sabbatical. :sweat_smile:

File: src/packagemanager.hpp [hunk 1]
```diff
@@ -12,8 +12,8 @@ NOTE: Spaces are VERY important because we need them to append other strings to
 */
 
 struct PackageManager {
-	std::string search, install, uninstall, autoremove, update, upgrade, upgrade_pkg, clean;
+	std::string search, install, uninstall, autoremove, update, upgrade, upgrade_pkg, clean, layman;
```
Comment (cvengler) [inline src/packagemanager.hpp]: I would not add `layman` as an option supported for all package manager. The Gentoo packaging system is to my knowledge to only one that supports it. People could invoke it natively anyway.

**Review by cvengler — CHANGES_REQUESTED** [review]

**Comment by swagtoy** [comment]
Seems good besides the layman options.

File: src/packagemanager.hpp (hunk 1 above)
Comment (td5038) [reply src/packagemanager.hpp]: I just made the changes

**Pull request merged**